src/components/ResultTable.js

```diff
@@ -9,15 +9,33 @@ function calculateOverallStrength(cells) {
 }
 
 class ResultTable extends React.Component {
+	constructor(props) {
```

Comment:
You don't need to add this since it is automatically done by React, we only need to add explicit constructor when there's some need of initialising other class members.